### ISSUE TITLE
Content files gets deployed twice

### ISSUE THREAD

**Issue by dotMorten** [NuGet/Home#6432]
When I add the following to my nuget package:
`/contentFiles/any/uap10.0/MyFolder/MyFile.txt`
And add this entry to the nuspec:

```xml
    <contentFiles>
      <files include="**/*.*" buildAction="Content" copyToOutput="true" flatten="false" />
    </contentFiles>
```

and then reference the nuget from a UWP app, when you build and deploy the app, in the generated appx folder you'll find "MyFile.txt" in the root folder and also in the /MyFolder/ folder.

The workaround is to set the `buildAction` to `none`, but that seems very counter intuitive. I would have expected that setting `copyToOutput` to `false` would have been the fix, because it gets deployed as content instead, and `flatten` is set to false., but it seems that `buildAction="content"` will deploy the files to the root folder, and `copyToOutput="true" flatten="false"` will _also_ deploy the files to the subfolder.

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): VS UI

NuGet version (x.x.x.xxx): 4.5.0.4685

VS version (if appropriate): 15.5.2

OS version (i.e. win10 v1607 (14393.321)): 10.0.16299.125

@emgarten

**Comment by nkolev92** [comment]
I'd imagine this is by design from UWP side, as Nuget just writes out the content files and the attributes, and then msbuild handles the rest. 

Need to repro it to confirm. 

The build log /v:d should help us see what happens.

**Comment by nkolev92** [comment]
Looking at the log files, it's kind of confusing and without insight into how the build work.
As far as NuGet is concerned, we do the right thing during restore.

I can tell that the copying is done here 
```
Target "_CopyFilesMarkedCopyLocal" in file "C:\Program Files (x86)\Microsoft Visual Studio\Preview\Enterprise\MSBuild\15.0\bin\Microsoft.Common.CurrentVersion.targets" from project "C:\Users\nikolev.REDMOND\Source\Repos\Library\App1\App1.csproj" (target "CopyFilesToOutputDirectory" depends on it):
Using "Copy" task from assembly "Microsoft.Build.Tasks.Core, Version=15.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
Task "Copy"
.
.
.
  Copying file from "C:\Users\nikolev.REDMOND\.nuget\packages\library\1.0.0\contentfiles\any\uap10.0\subfolder\SubfolderContent.txt" to "bin\x86\Debug\subfolder\SubfolderContent.txt".

```
That's the one that copies it correctly. 

Now the 2nd one, not sure where it's coming from.

Notable, file is in 2 different output groups
```
  App1 : App1.CopyLocalFilesOutputGroup : C:\Users\nikolev.REDMOND\.nuget\packages\library\1.0.0\contentfiles\any\uap10.0\subfolder\SubfolderContent.txt -> subfolder\SubfolderContent.txt

  App1 : App1.ContentFilesProjectOutputGroup : C:\Users\nikolev.REDMOND\.nuget\packages\library\1.0.0\contentfiles\any\uap10.0\subfolder\SubfolderContent.txt -> SubfolderContent.txt
```

tl;dr; 

This is happening in the build tasks.
